web/frontend/src/components/board/index.svelte

```diff
@@ -95,7 +95,7 @@ const writeAnalog = async () => {
 
 const readAnalog = async () => {
   try {
-    const value = await boardClient.readAnalogReader(analogPinName);
+    const value = await boardClient.readAnalogReadcd .er(analogPinName);
```

Comment:
This doesn't look right - what's going on here?

Reply:
an unfortunate typo when I was looking for mentions of analog reader in rdk.

Reply:
reverted :)

Reply:
Thanks! Removing myself because there are no more RC changes :)